### PR TITLE
Add voting power to balance

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: sdk/src/wallet/account/operations/balance.rs [hunk 1]
```diff
@@ -281,15 +281,18 @@ impl AccountHandle {
...
         Ok(AccountBalance {
             base_coin: BaseCoinBalance {
                 total: total_amount,
                 #[cfg(not(feature = "participation"))]
                 available: total_amount.saturating_sub(locked_amount),
                 #[cfg(feature = "participation")]
-                available: total_amount
-                    .saturating_sub(locked_amount)
-                    .saturating_sub(self.get_voting_power().await?),
+                available: total_amount.saturating_sub(locked_amount).saturating_sub(voting_power),
```
Comment (Thoralf-M) [inline sdk/src/wallet/account/operations/balance.rs:293]:
```suggestion
                available: total_amount.saturating_sub(locked_amount+voting_power),
```

Comment (thibault-martinez) [reply sdk/src/wallet/account/operations/balance.rs:293]: No risk of overflowing the addition? Should be fine I guess

Comment (Thoralf-M) [reply sdk/src/wallet/account/operations/balance.rs:293]: There is probably a risk if there is a larger token supply

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/src/wallet/bindings/nodejs/types/account.ts [hunk 2]
```diff
@@ -40,6 +40,8 @@ export interface BaseCoinBalance {
     total: string;
     /** The amount of the outputs that aren't used in a transaction */
     available: string;
+    /** Voting power */
+    voting_power: string;
```
Comment (Tuditi) [inline sdk/src/wallet/bindings/nodejs/types/account.ts]:
```suggestion
    votingPower: string;
```

File: sdk/src/wallet/bindings/java/lib/src/main/java/org/iota/types/BaseCoinBalance.java [hunk 3]
```diff
@@ -36,14 +42,16 @@ public BaseCoinBalance deserialize(final JsonElement json, final Type typeOfT, f
 
         long total = Long.parseLong(json.getAsJsonObject().get("total").getAsString());
         long available = Long.parseLong(json.getAsJsonObject().get("available").getAsString());
+        long voting_power = Long.parseLong(json.getAsJsonObject().get("votingPower").getAsString());
 
-        return new BaseCoinBalance(total, available);
+        return new BaseCoinBalance(total, available, voting_power);
```
Comment (Tuditi) [inline sdk/src/wallet/bindings/java/lib/src/main/java/org/iota/types/BaseCoinBalance.java]: Afaik camelCase is also the convention in Java.

**Review by Tuditi — CHANGES_REQUESTED** [review]
Are you still planning to keep the `getVotingPower()` function then?

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by Tuditi — APPROVED** [review]
Works fine in the nodejs examples! :fire:

**Pull request merged**